source/fab/parse/fortran.py

```diff
@@ -185,7 +198,7 @@ def walk_nodes(self, fpath, file_hash, node_tree) -> AnalysedFortran:
                         analysed_fortran.add_symbol_dep(called_name.string)
 
                 elif obj_type == Program_Stmt:
-                    analysed_fortran.add_symbol_def(str(obj.get_name()))
+                    analysed_fortran.add_program_def(str(obj.get_name()))
```

Comment:
This change might also fix the "unused symbols" check incorrectly listing program symbols. See #219.

Reply:
Sadly not, as it calls `add_symbol_def` itself. Should still be an easy fix when we come to it though.